open_lm/train.py

```diff
@@ -88,7 +88,7 @@ def sample_chunk(chunk, args):
     if chunk.shape[1] == args.seq_len + 1:
         start_idx = 0
     elif chunk.shape[1] > args.seq_len + 1:
-        start_idx = torch.randint(0, chunk.shape[1] - args.seq_len + 1, (1,)).item()
+        start_idx = torch.randint(0, chunk.shape[1] - args.seq_len - 1, (1,)).item()
```

Comment:
I think it should be `torch.randint(0, chunk.shape[1] - args.seq_len, (1,)).item()`

Reply:
Oh yeah you're right the high parameter is one above the largest number to be drawn.